feat(17821): Add support for "persistence" capability

vanch3d opened the pull request:
See https://hivemq.kanbanize.com/ctrl_board/57/cards/17821/details/

This PR creates the frontend infrastructure required to handle broker capabilities in general (handling the API, verification of a capability) and "persistence" capability.

It adds a tab to the bridge configuration panel, if and only if the `mqtt-persistence` is defined. The content is based on the newly added `persist` property of the `Bridge`. 

### Out-of-scope
- The is no notification to the user when the capability is not available (error or not added to broker). 


### Before
![screenshot-localhost_3000-2023 11 28-15_26_47](https://github.com/hivemq/hivemq-edge/assets/2743481/4a06bb33-7a40-4e9b-be2d-22f982b1aac8)

### After
![screenshot-localhost_3000-2023 12 04-13_12_19](https://github.com/hivemq/hivemq-edge/assets/2743481/368d34fa-f7de-45dd-94b9-1446d35b5f7f)

sfrehse commented:
Thanks. I guess we need to shorten the tab titles, however, let's sync with @DC2-DanielKrueger and Owen on that.

vanch3d commented:
Refactored the translations as per discussion and added a restart warning, activated on after change to the checkbox

![screenshot-localhost_3000-2023 12 05-17_51_00](https://github.com/hivemq/hivemq-edge/assets/2743481/0655ade9-c21d-42d8-bffa-b2f2254482a1)
